test/ducks/apps/components/applicationPage/__snapshots__/details.spec.js.snap

```diff
@@ -463,6 +452,16 @@ exports[`ApplicationPage details component should handle correctly \`display mor
         parseEmoji={true}
         source="### Features
 
+Enjoy features to grab your data in you Cozy :tada:### Features
+
+Enjoy features to grab your data in you Cozy :tada:### Features
+
+Enjoy features to grab your data in you Cozy :tada:### Features
+
+Enjoy features to grab your data in you Cozy :tada:### Features
+
+Enjoy features to grab your data in you Cozy :tada:### Features
+
```

Comment:
Is it expected to get five repetitions here ?

Reply:
Yep, for the test I just duplicated the text to reproduce a long text with new lines.